.yamato/config/universal.metafile

```diff
@@ -22,8 +27,7 @@ platforms:
       - name: DX12
       - name: Vulkan
     build_configs:
-      - name: il2cpp_apiNet4
```

Comment:
Is removing some configurations (either il2cpp or mono depending on the platform) intentional? I think we never tested both anyway but just want to confirm.

Reply:
yes, we can enable them later, but we dont want to test them right now

Reply:
(theyre in the shared metafile)